From a362668ac7ca891974c3b5b50c983471e1dfab02 Mon Sep 17 00:00:00 2001
From: Douglas Henrique Koerich <douglashenrique.koerich@windriver.com>
Date: Wed, 18 Aug 2021 12:19:16 -0400
Subject: [PATCH] Customize fpga_5gnr config for 1 VF

Change distribution of UL/DL queues to be used by one VF only.

Signed-off-by: Douglas Henrique Koerich <douglashenrique.koerich@windriver.com>
---
 fpga_5gnr/fpga_5gnr_config_vf.cfg | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/fpga_5gnr/fpga_5gnr_config_vf.cfg b/fpga_5gnr/fpga_5gnr_config_vf.cfg
index 0a82f52..81aa3bb 100644
--- a/fpga_5gnr/fpga_5gnr_config_vf.cfg
+++ b/fpga_5gnr/fpga_5gnr_config_vf.cfg
@@ -7,9 +7,9 @@ pf_mode_en = 0
 [UL]
 bandwidth = 3
 load_balance = 128
-vfqmap = 16,16,0,0,0,0,0,0
+vfqmap = 32,0,0,0,0,0,0,0
 
 [DL]
 bandwidth = 3
 load_balance = 128
-vfqmap = 16,16,0,0,0,0,0,0
+vfqmap = 32,0,0,0,0,0,0,0
-- 
2.29.2

